extensions/hibernate-search-standalone-elasticsearch/deployment/src/main/java/io/quarkus/hibernate/search/standalone/elasticsearch/deployment/dev/HibernateSearchStandaloneDevUIProcessor.java

```diff
@@ -0,0 +1,56 @@
+package io.quarkus.hibernate.search.standalone.elasticsearch.deployment.dev;
+
+import static io.quarkus.deployment.annotations.ExecutionTime.RUNTIME_INIT;
+
+import java.util.Optional;
+
+import io.quarkus.arc.deployment.AdditionalBeanBuildItem;
+import io.quarkus.arc.processor.DotNames;
+import io.quarkus.deployment.IsDevelopment;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.annotations.BuildSteps;
+import io.quarkus.deployment.annotations.Record;
+import io.quarkus.devui.spi.JsonRPCProvidersBuildItem;
+import io.quarkus.devui.spi.page.CardPageBuildItem;
+import io.quarkus.devui.spi.page.Page;
+import io.quarkus.hibernate.search.standalone.elasticsearch.deployment.HibernateSearchStandaloneEnabled;
+import io.quarkus.hibernate.search.standalone.elasticsearch.deployment.HibernateSearchStandaloneEnabledBuildItem;
+import io.quarkus.hibernate.search.standalone.elasticsearch.runtime.HibernateSearchStandaloneRuntimeConfig;
+import io.quarkus.hibernate.search.standalone.elasticsearch.runtime.dev.HibernateSearchStandaloneDevJsonRpcService;
+import io.quarkus.hibernate.search.standalone.elasticsearch.runtime.dev.HibernateSearchStandaloneDevRecorder;
+
+@BuildSteps(onlyIf = { HibernateSearchStandaloneEnabled.class, IsDevelopment.class })
+public class HibernateSearchStandaloneDevUIProcessor {
+
+    @BuildStep
+    @Record(RUNTIME_INIT)
+    public CardPageBuildItem create(HibernateSearchStandaloneDevRecorder recorder,
+            HibernateSearchStandaloneRuntimeConfig runtimeConfig,
+            Optional<HibernateSearchStandaloneEnabledBuildItem> enabled) {
+        recorder.initController(enabled.isPresent(), runtimeConfig);
+
+        CardPageBuildItem card = new CardPageBuildItem();
+        card.addPage(Page.webComponentPageBuilder()
+                .title("Indexed Entity Types")
```

Comment:
Due to common APIs and error messages, we do have to use the name entity even in this context. It even makes sense, though I'll admit it may not be very intuitive.

We're consistent and try to document this, though: users need to annotate their classes with `@SearchEntity`, and the guide links to that part of the reference documentation that explains what we mean by "entity": https://docs.jboss.org/hibernate/stable/search/reference/en-US/html_single/#concepts-entity

I hope we can make this clearer in the future, but for now I think we'll have to settle for this.